src/main/resources/templates/index.html

```diff
@@ -264,9 +266,5 @@ <h2>Mood Entry</h2>
 <!--    </div>-->
 </div>
 
-
-<script type="text/javascript">
-    init();
-</script>
```

Comment:
Ditto - "Not a best practice to call a function without the document being ready and without context. The jquery function "$(document).ready()" serves this purpose, thus reducing technical debt."